enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -188,6 +196,13 @@ private void applyQueuedTransactions() throws IOException
             // Check rotation explicitly, since the version of append that we're calling isn't doing that.
             logRotation.rotateLogIfNeeded( LogAppendEvent.NULL );
 
+            // Check kernel health after log rotation
+            if ( !kernelHealth.isHealthy() )
+            {
+                log.error( msg );
```

Comment:
we still have 'causeOfPanic' in kernelHealth so we can log actual problem.
And yes it's true that we log it initially, but i would assume as soon as we become unhealthy we will log plenty of messages into log file and it will be hard to find initial cause, if not impossible because we can rotate log file and delete it.